include/opendht/indexation/pht.h

```diff
@@ -115,6 +110,27 @@ struct Prefix {
         return 8 * i + j;
     }
 
+    /**
+     * This method swap the bit a the position 'bit' and return the new prefix
+     *
+     * @param bit Position of the bit to swap
+     *
+     * @return The prefix with the bit at position 'bit' swapped
+     *
+     * @throw out_of_range Throw out of range if bit does not exist
+     */
+    Prefix swapBit(size_t bit) const {
+	if ( bit >= copy.size_ ) 
```

Comment:
Just saw that right now, there is a tabulation here and line just under this one
Line 123 :  \tif( bit >= copy.size_)
Line 124 :  \t    throw std::out_of_range("bit larger than prefix size.");

Could you please edit that ? 
Or should i do it and then give you the commit to cherry pick ?